.gitignore

```diff
@@ -23,3 +23,4 @@ aula.yaml*
 /static/avatars/*.png
 /hpc
 cookie-jar
+src/Backend.hs
```

Comment:
nope, we still want this file to stick around...  if you have local changes, you just need to add around them manually.  (i use git-cola for that, but i always suspected there are better tools, too.)